src/open_inwoner/accounts/tests/factories.py

```diff
@@ -58,6 +58,7 @@ class Meta:
 
     name = factory.Faker("first_name")
     created_by = factory.SubFactory(UserFactory)
+    is_deleted = False
```

Comment:
Is this required?Since it is False by default.

Reply:
Hmm.. not really. I'll remove it.